subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/ArtifactDependencies.java

```diff
@@ -39,5 +39,5 @@
 @Target({ElementType.METHOD})
 @Documented
 @InjectionPointQualifier(supportedTypes = FileCollection.class)
-public @interface PrimaryInputDependencies {
+public @interface ArtifactDependencies {
```

Comment:
Hm, I thought we wanted to call this `@InputArtifactDependencies`?

Reply:
Fine with me as well. I used https://docs.google.com/spreadsheets/d/1j5FCYWRvMeYoAjH4MYz7hB-r54i-9I0XrSxdT4FCHJQ/edit#gid=0 as base for the renames.

Reply:
I think we should go with `@InputArtifactDependencies`, and have all input annotations' name start with `@Input`.

Reply:
This rename has been done as part of #8510.